### PR TITLE
feat(alert-preview): show last triggered in preview

### PR DESCRIPTION
**Pull request by Zandew**
Display a column for last triggered in preview table.

Screen Shot 2022-12-06 at 12 45 27 PM

Also adds a tooltip describing which previews are approximate.

Screen Shot 2022-12-06 at 1 06 35 PM

### CODE REVIEW

File: static/app/components/issues/groupListHeader.tsx [hunk 1]
```diff
@@ -7,15 +7,21 @@ import space from 'sentry/styles/space';
...
+  narrowGroups = false,
+  showLastTriggered = false,
+}: Props) => (
   <PanelHeader disablePadding>
     <IssueWrapper>{t('Issue')}</IssueWrapper>
     {withChart && <ChartWrapper narrowGroups={narrowGroups}>{t('Graph')}</ChartWrapper>}
     <EventUserWrapper>{t('events')}</EventUserWrapper>
     <EventUserWrapper>{t('users')}</EventUserWrapper>
     <AssigneeWrapper narrowGroups={narrowGroups}>{t('Assignee')}</AssigneeWrapper>
+    {showLastTriggered && <EventUserWrapper>{t('last triggered')}</EventUserWrapper>}
```
Comment (scttcper) [inline static/app/components/issues/groupListHeader.tsx:24]:
```suggestion
    {showLastTriggered && <EventUserWrapper>{t('Last Triggered')}</EventUserWrapper>}
```

File: static/app/components/stream/group.tsx [hunk 2]
```diff
@@ -415,6 +419,33 @@ function BaseGroupRow({
...
+                  <TimeSince
+                    tooltipTitle={t('Last Triggered')}
+                    date={lastTriggeredDate}
+                    suffix={t('ago')}
+                    shorten
+                  />
+                </div>
+              </span>
+            </span>
+          </GuideAnchor>
```
Comment (scttcper) [inline static/app/components/stream/group.tsx:443]: do you need this to be a dropdown? Copied guide anchor

File: static/app/views/alerts/rules/issue/index.tsx [hunk 3]
```diff
@@ -937,9 +938,17 @@ class IssueRuleEditor extends AsyncView<Props, State> {
       );
     }
     return tct(
-      "[issueCount] issues would have triggered this rule in the past 14 days approximately. If you're looking to reduce noise then make sure to [link:read the docs].",
+      "[issueCount] issues would have triggered this rule in the past 14 days [approximately]. If you're looking to reduce noise then make sure to [link:read the docs].",
```
Comment (scttcper) [inline static/app/views/alerts/rules/issue/index.tsx:941]:
```suggestion
      "[issueCount] issues would have triggered this rule in the past 14 days [approximately:approximately]. If you're looking to reduce noise then make sure to [link:read the docs].",
```

File: static/app/views/alerts/rules/issue/index.tsx [hunk 4]
```diff
@@ -937,9 +938,17 @@ class IssueRuleEditor extends AsyncView<Props, State> {
...
       {
         issueCount,
+        approximately: (
+          <Tooltip
+            title="Previews that include issue frequency conditions are approximated"
+            showUnderline
+          >
+            approximately
+          </Tooltip>
```
Comment (scttcper) [inline static/app/views/alerts/rules/issue/index.tsx:950]:
```suggestion
          <Tooltip
            title="Previews that include issue frequency conditions are approximated"
            showUnderline
          />
```

File: static/app/views/alerts/rules/issue/index.tsx (hunk 3 above)
Comment (scttcper) [reply static/app/views/alerts/rules/issue/index.tsx:941]: add the text inline so the translation has the entire text content

File: static/app/types/group.tsx [hunk 5]
```diff
@@ -430,6 +430,7 @@ interface GroupFiltered {
 export interface GroupStats extends GroupFiltered {
   filtered: GroupFiltered | null;
   id: string;
+  lastTriggered?: string;
```
Comment (scttcper) [inline static/app/types/group.tsx:433]: this needs a comment

File: static/app/components/stream/group.tsx [hunk 6]
```diff
@@ -415,6 +419,17 @@ function BaseGroupRow({
     </DeprecatedDropdownMenu>
   );
 
+  const lastTriggered = !defined(lastTriggeredDate) ? (
+    <Placeholder height="18px" />
```
Comment (scttcper) [inline static/app/components/stream/group.tsx:423]: does this need a loading placeholder or is the stat available as soon as it renders?

**Review by scttcper — APPROVED** [review]

File: static/app/components/stream/group.tsx (hunk 6 above)
Comment (Zandew) [reply static/app/components/stream/group.tsx:423]: It should be available when it renders, it's returned from the same endpoint as the groups themselves

**Pull request merged**